react/features/app/components/App.native.js

```diff
@@ -11,6 +11,7 @@ import '../../mobile/full-screen';
 import '../../mobile/permissions';
 import '../../mobile/proximity';
 import '../../mobile/wake-lock';
+import '../../analytics';
```

Comment:
isn't this imported anywhere? if it isn't, sort it alphabetically

Reply:
Fixed it.